script.js

```diff
@@ -1,13 +1,14 @@
 
 // Config
-var latitude = "40.758701";
-var longitude = "-111.876183";
+var latitude = localStorage.getItem("latitude");
```

Comment:
@Evermuniz I couldn't find  localstorage.setItem for latitude, longitude. Just take a look on it.